docs/configuration.md

```diff
@@ -886,6 +892,17 @@ and proxy connections to the registry server.
        Absolute path to x509 cert file
     </td>
   </tr>
+  <tr>
+    <td>
+      <code>rootcas</code>
+    </td>
+    <td>
+      yes
+    </td>
+    <td>
+      An array of paths to the root CA files, as x509.
```

Comment:
"An array of paths to root CA files in x509 format used to verify client certificates."